web/src/components/tasks/RaffleBingoPaginator.test.tsx

```diff
@@ -0,0 +1,146 @@
+    real task in the context of a test environment.
+  */
+  const tabTask = await import(`@businessnjgovnavigator/content/roadmaps/raffle-bingo-steps/${filename}.md`);
+
+  return {
+    ...convertTaskMd(tabTask.default),
+    unlockedBy: [],
+    filename,
+  };
+};
+
+let step1: Task;
+let step2: Task;
+
+describe("<RaffleBingoPaginator />", () => {
+  beforeEach(async () => {
+    jest.resetAllMocks();
+    useMockRoadmap({});
+    useMockRouter({});
+    setupStatefulUserDataContext();
+
+    step1 = await fetchRaffleBingoTab("raffle-license-step-1");
+    step2 = await fetchRaffleBingoTab("raffle-license-step-2");
+  });
+
+  const task = generateTask({});
+
+  const renderPaginator = (task: Task): void => {
+    render(
+      <WithStatefulUserData initialUserData={generateUserData({})}>
+        <RaffleBingoPaginator task={task} />
+      </WithStatefulUserData>
+    );
+  };
+
+  it("renders first tab content when the task is opened", async () => {
+    renderPaginator(task);
+
+    await waitFor(() => {
+      expect(screen.getByText("Eligibility Criteria")).toBeInTheDocument();
+    });
+
+    expect(screen.getByRole("button", { name: step1.callToActionText })).toBeInTheDocument();
+    expect(screen.getByRole("button", { name: Config.taskDefaults.continueButtonText })).toBeInTheDocument();
+    expect(
+      screen.queryByRole("button", { name: Config.taskDefaults.backButtonText })
+    ).not.toBeInTheDocument();
+    expect(
+      screen.queryByRole("button", { name: Config.taskDefaults.markAsCompleteButtonText })
+    ).not.toBeInTheDocument();
+  });
+
+  it("displays second step task content when on second step", async () => {
+    renderPaginator(task);
+
+    await screen.findByTestId("stepper-1");
+
+    fireEvent.click(screen.getByTestId("stepper-1"));
+
+    await waitFor(() => {
+      expect(screen.getByText("Application Requirements")).toBeInTheDocument();
+    });
+
+    await waitFor(() => {
+      expect(screen.getByRole("button", { name: step2.callToActionText })).toBeInTheDocument();
+    });
+
+    expect(screen.getByRole("button", { name: Config.taskDefaults.backButtonText })).toBeInTheDocument();
+    expect(
+      screen.queryByRole("button", { name: Config.taskDefaults.continueButtonText })
+    ).not.toBeInTheDocument();
+  });
+
+  it("marks task as complete when 'Mark As Complete' button is clicked", async () => {
+    renderPaginator(task);
+
+    await screen.findByTestId("stepper-1");
+
+    fireEvent.click(screen.getByTestId("stepper-1"));
+
+    await waitFor(() => {
+      expect(screen.queryByText(getTaskStatusUpdatedMessage("COMPLETED"))).not.toBeInTheDocument();
+    });
+
+    fireEvent.click(screen.getByRole("button", { name: Config.taskDefaults.markAsCompleteButtonText }));
+
+    await screen.findByText(getTaskStatusUpdatedMessage("COMPLETED"));
+    expect(screen.getByText(getTaskStatusUpdatedMessage("COMPLETED"))).toBeInTheDocument();
```

Comment:
[pebble] Aren't these 2 lines doing almost the same thing? Why not simplify this?
```suggestion
    expect(await screen.findByText(getTaskStatusUpdatedMessage("COMPLETED"))).toBeInTheDocument();

```